docs/guides.md

```diff
@@ -484,6 +500,546 @@ and refresh rate prior to connecting to an app. See
+
+@note{On a default Debian 12 install using KDE Plasma, you are using the Simple Desktop Display Manager (SDDM).
+Even if you are logging in to a wayland session, SDDM by default starts with an xorg session, so this script
+does not need to be modified if you primarily use a wayland session (the default) when you login}
+
+#### Instructions
+
+##### Enable Wake on LAN
+
+Wake on LAN (WOL) will allow you to send a magic packet to turn your PC on remotely. This is handled automatically by
+Moonlight's "send wake on lan" option in the app but you do need to enable it on your host machine first. The
+[instructions on the debian.org](https://wiki.debian.org/WakeOnLan#Enabling_WOL) site are a little hard to parse, so
+I've simplified them below.
+
+@note{this may not work on all deb based distributions. If you know of a better way for POP OS, Ubuntu, or another
+debian based distro please feel free to edit the guide yourself, or let me know.}
+
+First, find the name of your ethernet interface.
+
+```bash
+ip link show
+```
+
+When I run this command, these are the results I receive
+```
+rodent@midwest:/etc/network/if-up.d$ ip link show  
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000  
+   link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00  
+2: enp117s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP mode DEFAULT group default qlen 1000  
+   link/ether 9c:6b:00:59:33:c1 brd ff:ff:ff:ff:ff:ff
+```
+
+We can ignore the loopback interface, and I can see my ethernet interface is called `enp117s0`. You might see
+wireless interfaces here as well but they can also be ignored.
+
+@note{If your PC is only connected via wifi, it is still technically possible to get this working, but it is outside
+the scope of this guide and requires more networking knowledge and a wifi chip that supports WOL. If this is your
+first foray into linux, I'd recommend just getting a cable.}
+
+Now I can install ethtool and modify my interface to allow wake on lan. For your use, replace enp117s0 with whatever
+the name of your ethernet interface is from the command `ip link show`
+
+``` bash
+sudo apt update
+sudo apt install ethtool
+sudo ethtool -s enp117s0 wol g
+```
+##### SSH Client Setup
+To start, we need to install an SSH client on our machine if this not already done. Open a terminal and enter the
+following command.
+
+```bash
+sudo apt update
+sudo apt install openssh-client
+```
+
+Next we need to generate the keys we will use to connect to our SSH session. This is as simple as running the
+following in a terminal:
+
+```bash
+ssh-keygen
+```
+
+and simply pressing enter through the default options. This will place a file called id_rsa and id_rsa.pub
+in the hidden folder ~/.ssh/. This is the default key used when this user initiates an SSH session.
```

Comment:
```suggestion
and simply pressing enter through the default options. This will place a file called `id_rsa` and `id_rsa.pub`
in the hidden folder `~/.ssh/`. This is the default key used when this user initiates an SSH session.
```